app/controllers/home_controller.rb

```diff
@@ -5,15 +5,11 @@ class HomeController < ApplicationController
   def index
     @projects =
       Rails.cache.fetch("home-index", expires_in: 24.hours) do
-        _projects =
-          [
-            Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
-            Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
-            Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
-          ]
-        _projects.map! { |project| ProjectDecorator.new(project) }
-        _projects.each(&:process_with)
-        _projects
+        [
+          Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
+          Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
+          Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
+        ].map! { |project| ProjectDecorator.new(project) }.each(&:process_with)
```

Comment:
Correct me if I am wrong.. The cache would cache the last return of this, which is `each(&:process_with)` and I believe that only returns a status of `true` or `nil` so it's not exactly an array of `ProjectDectorator`s anymore.